site/package.json

```diff
@@ -3,8 +3,11 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "copy:pure": "cp -r ../build/ ./static/css/pure/",
```

Comment:
Not sure I get the point here? Why the extra copy or how is it better than before?

Reply:
I admit this is funky and sort of a hack to work around docusaurus. I want to use the locally built version of pure so that the layout examples are always using the `master` version, previously they used a hardcoded CDN link.

To get docusaurus to register the css as static they have to exist in the static/ directory before the site is built. Therefore, I just copy them in manually and gitignore them. Once docusaurus builds, they resolve as CSS files properly.

Reply:
This could a useful documentation note somewhere. I think we can't put comments inside `package.json`..

Reply:
Good call. I can put it in a readme under the site directory.

Reply:
Added https://github.com/pure-css/pure/pull/790/commits/5d3fa31dc54171b408f558e7ca846091541049cd